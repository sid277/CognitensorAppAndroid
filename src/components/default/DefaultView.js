import React from 'react';
import { View, StyleSheet } from 'react-native';

import { theme } from '../../theme';

const styles = StyleSheet.create({
  container: {
    flex: 1,
    backgroundColor: theme.colors.white,
  },
});

const DefaultView = ({ children, styleView }) => {
  return <View style={[styles.container, styleView]}>{children}</View>;
};

export default DefaultView;
