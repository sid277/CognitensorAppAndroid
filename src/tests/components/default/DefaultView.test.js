import React from 'react';
import { shallow } from 'enzyme';
import DefaultView from '../../../components/default/DefaultView';
import renderer from 'react-test-renderer';


test('should test DefaultScrollView component', () => {
const wrapper = shallow(<DefaultView/>);
expect(wrapper).toMatchSnapshot();
});
