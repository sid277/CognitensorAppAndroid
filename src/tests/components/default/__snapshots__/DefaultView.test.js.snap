// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should test DefaultScrollView component 1`] = `ShallowWrapper {}`;
